website_sale_require_legal/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+website
```

Comment:
This file goes in the repo root, not in the addon folder.

Reply:
Right! 😅